settings/provisioner/openstack/openstack.spec

```diff
@@ -0,0 +1,49 @@
+---
+subparsers:
+    openstack:
+        help: Provision systems using Ansible OpenStack modules
+        groups:
+            - title: image
+              options:
+                  image-name:
+                      type: Value
+                      help: An image on OpenStack cloud to provision the instance with
+                      required: yes
+            - title: topology
+              options:
+                  neutron:
+                      type: YamlFile
+                      help: Network resources
+                      default: default.yml
+                  topology:
```

Comment:
this will map to `provisioner.topology.topolgy` I suggest to rename this to `nodes`, or, `nova`